packages/react-widgets/src/DateTimePicker.js

```diff
@@ -361,7 +363,7 @@ class DateTimePicker extends React.Component {
         autoFocus={autoFocus}
         placeholder={placeholder}
         disabled={disabled}
-        readOnly={readOnly}
+        readOnly={readOnly || inputReadOnly}
```

Comment:
I think this should be `inputReadOnly == null ? readOnly : inputReadOnly` that way the inputReadOnly is always used if it's set to false/true

Reply:
Ah right @jquense, I'd done it the other way 😅. That's `inputReadOnly || readOnly` then - will adjust the PR

Reply:
quick comment before you do: i think `inputReadOnly || readOnly` is not quite right, since it will fall back for `false`, the idea is you want it to always use `true` or `false` and only fallback when its `null` or `undefined` (e.g. not added)

Reply:
🤦‍♂️ absolutely correct @jquense, my bad. Should be good to go now!